src/transformers/models/bart/tokenization_bart.py

```diff
@@ -318,7 +318,7 @@ def save_vocabulary(self, save_directory: str, filename_prefix: Optional[str] =
         )
 
         with open(vocab_file, "w", encoding="utf-8") as f:
-            f.write(json.dumps(self.encoder, ensure_ascii=False))
+            f.write(json.dumps(self.encoder, indent=2, sort_keys=True, ensure_ascii=False) + "\n")
```

Comment:
are we sure we want to `sort_keys`?

Reply:
consistency with already published models

Reply:
The model's configs are already sorted: https://huggingface.co/facebook/opt-350m/blob/main/config.json but it's true it would be more consistent with existing tokenizer configs. Think I'd still be in favor of sorting here though

Reply:
ok, I didn't remember configs were already sorted. Sounds good to me then!